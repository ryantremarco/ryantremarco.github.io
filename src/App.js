import React from 'react';
import { HashRouter as Router, Route, Switch } from 'react-router-dom';
import Home from "./pages/Home";
import NotFound from './pages/Home/NotFound';

const App = () => {
  const classes = {
    app: {
      display: "flex",
      minHeight: "100vh",
      flexDirection: "column",
    }
  }

  return (
    <div style={classes.app}>
      <Router basename="/">
        <Switch>
          <Route exact path="/" component={Home}/>
          <Route path="/home" component={Home}/>
          <Route component={NotFound}/>
        </Switch>
      </Router>
    </div>
  );
}

export default App;
