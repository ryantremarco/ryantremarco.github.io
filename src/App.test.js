import renderer from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  const tree = renderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();
});
