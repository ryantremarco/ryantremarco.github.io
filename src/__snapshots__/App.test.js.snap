// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "minHeight": "100vh",
    }
  }
>
  <nav
    className="navbar"
  >
    <div
      className="navbar-brand"
    >
      <a
        className="navbar-item"
        href="#/"
        onClick={[Function]}
      >
        <h1
          className="title"
        >
          Ryan Tremarco
        </h1>
      </a>
    </div>
    <div
      className="navbar-start"
    >
      <a
        className="navbar-item"
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="navbar-item"
        href="#/about"
        onClick={[Function]}
      >
        About Me
      </a>
      <a
        className="navbar-item"
        href="#/dev"
        onClick={[Function]}
      >
        Development
      </a>
      <a
        className="navbar-item"
        href="#/misc"
        onClick={[Function]}
      >
        Misc
      </a>
    </div>
    <div
      className="navbar-end"
    >
      <div
        class="navbar-item"
      >
        <div
          class="field is-grouped"
        >
          <p
            class="control"
          >
            <a
              class="button is-primary"
              href="#/contact"
              onClick={[Function]}
            >
              <span
                class="icon"
              >
                <i
                  class="fas fa-envelope"
                />
              </span>
              <span>
                Contact
              </span>
            </a>
          </p>
        </div>
      </div>
    </div>
  </nav>
  <section
    className="section"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <div
      className="container"
    >
      Content incoming :)
    </div>
  </section>
  <footer
    className="footer"
  >
    <div
      className="content has-text-centered"
    >
      Made with 
      <i
        class="fas fa-heart"
      />
       using
      <strong>
        <a
          href="https://bulma.io"
          target="_"
        >
           Bulma 
        </a>
      </strong>
      and
      <strong>
        <a
          href="https://reactjs.org"
          target="_"
        >
           React 
        </a>
      </strong>
    </div>
  </footer>
</div>
`;
